### ISSUE TITLE
Maybe invalid return from 'geocentricToGeodetic'

### ISSUE THREAD

**Issue by dmarteau** [proj4js/proj4js#425]
At https://github.com/proj4js/proj4js/blob/b0b33a3c7fc27d2e8de2b519f867838dda5a19c4/lib/datumUtils.js#L116, the return value shouldn't be 
```
      return {
        x: Longitude,
        y: Latitude,
        z: Height
      };
```
instead of 
```
      return {
        x: p.x,
        y: p.y,
        z: p.z
      };
```
?

**Comment by ahocevar** [comment]
The two are equivalent. Are you seeing an issue that makes you think there is something wrong here?

**Comment by dmarteau** [comment]
> Are you seeing an issue that makes you think there is something wrong here

Not really, but this was more a matter of code clarity there since variables `Latitude` and `Height` where set.

Note that Java port of proj4 return explicitely `Latitude` and `Height` instead of the original coordinates and I was wondering if that may have some side effects.
